pkgs/servers/tailscale/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitHub, makeWrapper, iptables, iproute }:
+
+buildGoModule rec {
+  pname = "tailscale";
+  version = "v0.97";
+
+  src = fetchFromGitHub {
+    owner = "tailscale";
+    repo = "tailscale";
+    rev = "v0.97";
```

```suggestion
  version = "0.97";

  src = fetchFromGitHub {
    owner = "tailscale";
    repo = "tailscale";
    rev = "v${version}";
```

nixpkgs manual says version shouldn't have that character in it.
Not sure if this commit is on master yet or not?

---

Woops, adding your suggested change resolved this.

To your other question: the version adjust isn't in master yet. I'll queue up a PR to do that as well. The version+rev change is a no-op on the built binaries though, since the v0.97 tag references the commit hash we used in the initial commit to unstable.